### PR TITLE
[GCU Documentation] Update Testing Description

### PR DESCRIPTION
**Pull request by isabelmsft**
Add links to GCU E2E tests in GCU testing documentation

### CODE REVIEW

File: doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md [hunk 1]
```diff
@@ -940,3 +925,29 @@ N/A
...
+| Test Case | Description |
+| --------- | ----------- |
+| 1         | [Updating Syslog configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_syslog.py) |
+| 2         | [Updating AAA configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_aaa.py) |
+| 3         | [Updating DHCP configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_dhcp_relay.py) |
+| 4         | [Updating IPv6 configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_ipv6.py) |
+| 5         | Updating monitor configs (EverflowAlaysOn). |
+| 6         | [Updating BGP speaker configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_bgp_speaker.py) |
+| 7         | [Updating BGP listener configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_bgpl.py) |
+| 8         | Updating Bounce back routing configs. |
```
Comment (qiluo-msft) [inline doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md:939]: also add link: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/bgp/test_bgp_bbr.py

File: doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md [hunk 2]
```diff
@@ -940,3 +925,29 @@ N/A
...
+| 9         | [Updating control-plane ACLs (NTP, SNMP, SSH) configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_cacl.py) |
+| 10        | [Updating Ethernet interfaces configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_eth_interface.py) |
+| 11        | [Updating VLAN interfaces configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_vlan_interface.py) |
+| 12        | [Updating port-channel interfaces configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_portchannel_interface.py) |
+| 13        | [Updating loopback interfaces configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_lo_interface.py) |
+| 14        | [Updating Kubernetes configs.](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_kubernetes_config.py) |
+| 15        | Updating BGP prefix hijack configs. |
```
Comment (qiluo-msft) [inline doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md:946]: @wen587 Do you know the link to this one?

File: doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md (hunk 1 above)
Comment (isabelmsft) [reply doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md]: added link

File: doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md (hunk 2 above)
Comment (wen587) [reply doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md:946]: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_bgp_prefix.py
Hi @isabelmsft , could you help add this  link.

Comment (isabelmsft) [reply doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md]: Thanks, added link

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**